requirements.txt

```diff
@@ -0,0 +1,17 @@
+numpy>=1.16.4
```

Comment:
Wazimap does not need a requirements file. Wazimap sets it's dependencies in its setup.py file.

Reply:
The requirements.txt way the way docker installed all the dependencies needed to run, it's not needed when published but helpful during development.